lib/exports/movie_widgets.dart

```diff
@@ -0,0 +1,10 @@
+library movie_widgets;
+export '../movie_widgets/fade_image_widget.dart';
+export '../movie_widgets/movie_detail_image.dart';
+export '../movie_widgets/movie_detail_play_button.dart';
+export '../movie_widgets/movie_details_actions.dart';
+export '../movie_widgets/movie_details_info.dart';
+export '../movie_widgets/movie_details_overview.dart';
+export '../movie_widgets/movie_details_title.dart';
+export '../movie_constants/movie_constants.dart';
+export '../movie_constants/movie_dimensions.dart';
```

Comment:
missed the extra line at the end, do Ctrl + Alt + L to fix it, or add it manually

Reply:
Done.